### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by janno42**
- sort minutes on minutes list descending by year
- use select2 fields for user selection in minutes form
- translations
- move comment field to bottom in document form
- don't show author in information documents anymore. see also #425

### CODE REVIEW

File: _1327/main/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -18,18 +18,164 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: admin.py:19
+#: _1327/main/forms.py:24
+msgid "You are only allowed to define a document"
+msgstr ""
```
Comment (Nef10) [inline _1327/main/locale/de/LC_MESSAGES/django.po]: warum gibt es hier fast keine Übersetzungen?

File: _1327/minutes/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3,87 +3,112 @@
...
+#: _1327/minutes/templates/minutes_meta_information.html:27
 msgid "Unpublished"
 msgstr "Unveröffentlicht"
 
-#: models.py:23
-msgid "Published"
-msgstr "Veröffentlicht"
+#: _1327/minutes/models.py:47
+msgid "Published for Students and University Network"
+msgstr ""
```
Comment (Nef10) [inline _1327/minutes/locale/de/LC_MESSAGES/django.po]: warum gibt es hier fast keine Übersetzungen?

File: _1327/user_management/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -18,70 +18,75 @@ msgstr ""
...
 
-#: forms.py:26
+#: _1327/user_management/forms.py:28
 msgid "This account is inactive."
 msgstr "Dieser Account ist deaktiviert."
 
-#: models.py:9
+#: _1327/user_management/forms.py:36
+msgid "Select a user"
+msgstr ""
```
Comment (Nef10) [inline _1327/user_management/locale/de/LC_MESSAGES/django.po]: warum fehlen in dieser Datei einige Übersetzungen?

**Comment by Nef10** [comment]
If adding an internal link in the markdown editor this happens with the new select2:

screenshot 2016-10-29 17 18 00

**Comment by janno42** [comment]
I will work on the translations again. There are some more missing from the other PR, so I would like to do that after both have been merged and keep the translations in this PR for now in their current state.

**Comment by janno42** [comment]
fixed select2

**Pull request merged**